site/docs/sql/intro.md

```diff
@@ -117,19 +124,19 @@ statement:
 )
 ```
 
-The SINK INTO command will look like this:
+The `SINK INTO` command will look like this:
 
 ```sql
 SINK INTO latest_trades(__key, this)
```

Comment:
Please replace `__key` and `this` with explicit references. It’s a default that’s not well known.